### DISCUSSION TITLE
Can't find info/docs on "attrs" attribute

### DISCUSSION THREAD

**Discussion by tsugabloom** [luwes/sinuous#180] [Q&A] [answered]
Hey Wesley, I noticed this code 

https://github.com/luwes/sinuous/blob/ece2c65c19939ec5742f999cfba6c83295c58118/packages/sinuous/h/src/property.js#L35

Can't find a real DOM attribute called `attrs` on MDN - is this Sinuous-specific? If so what is it for?

Cheers!

**Comment by luwes** [comment, marked as answer]
hey Gen, it's indeed specific, I either ported it from hyperscript or one of Ryan's implementations.

see here 
https://github.com/hyperhype/hyperscript/blob/master/index.js#L102

it's mainly because element properties are set by default, in some rare cases one might want to target element attributes directly.
